### ISSUE TITLE
Support for persistent 2.5

### ISSUE THREAD

**Issue by felixonmars** [prowdsponsor/esqueleto#137]
It would be nice to support the latest version of persistent 2.5 series.

**Comment by peti** [comment]
Yes, that would be very nice indeed! As it is now, `esqueleto` has effectively been removed from Stackage because it's blocking other packages. :cry:

**Comment by ilovezfs** [comment]
Any progress on this? Downstream is now starting to patch around this.

**Comment by mrkkrp** [comment]
Now that `lts-7.0` is out with GHC 8.0 we cannot switch to it solely because of `esqueleto`. @meteficha Any chance you take 5 minutes and merge a PR that addresses this?

**Comment by qrilka** [comment]
@meteficha was this project abandoned?

**Comment by wyager** [comment]
What's going on here? Looks like @vandenoever fixed it but didn't submit a PR to upstream Esqueleto?

**Comment by qrilka** [comment]
@wyager I've created alternative PR #153 but I will be happy to close it in favor of more proper one

**Comment by vandenoever** [comment]
@wyager It was @tebello-thejane  who fixed it for my use-case with https://github.com/prowdsponsor/esqueleto/pull/150. I cannot say which is the better PR.

**Comment by mrkkrp** [comment]
Maybe we should contact them at `opensource@prowdsponsor.com` and ask if this is maintained and if not, someone could take over it?

**Comment by clinty** [comment]
Perhaps an issue at https://github.com/haskell-infra/hackage-trustees/issues ?

**Comment by nd2s** [comment]
I suggest updating this ticket to be about `persistent-2.6` and `base-4.9.0.0`, because we have a `esqueleto-2.5.0` version now, but it still can't be use with `lts-7.x` because it's too old.

**Comment by ilovezfs** [comment]
@nd2s the new maintainer's repository is here: https://github.com/bitemyapp/esqueleto
